app/model/editions/EditionsIssue.scala

```diff
@@ -48,7 +48,8 @@ case class EditionsIssue(
     else fronts
       .filterNot(_.isHidden) // drop hidden fronts
       .map(_.toPublishedFront) // convert
-      .filterNot(_.collections.isEmpty) // drop fronts that contain no collections
+      .filterNot(_.collections.isEmpty), // drop fronts that contain no collections
+    EditionsTemplates.templates.get(edition).map(_.notificationUTCOffset).getOrElse(3)
```

Comment:
Could we add a comment to clarify why `3` or use a constant/`val` declared elsewhere (perhaps `DailyEdition.notificationUTCOffset`) for this?

Reply:
Yes, quite right, will do.  Magic numbers bad.